projects/packages/waf/src/class-waf-runner.php

```diff
@@ -292,31 +306,31 @@ public static function activate() {
 			Waf_Rules_Manager::generate_ip_rules();
 			self::create_blocklog_table();
 			Waf_Rules_Manager::generate_rules();
-		} catch ( \Exception $e ) {
-			return new WP_Error( 'waf_activation_failed', $e->getMessage() );
+		} catch ( Waf_Exception $e ) {
+			throw $e;
```

Comment:
We could also just not have the try-catch block here I think? (Again, cannot suggest cause you only edited these two lines)

Reply:
Resolved via https://github.com/Automattic/jetpack/commit/c1d3effd93c8fd23355781bca6b324383c18d498

Reply:
This one is also still there.

Reply:
Whoops [370a517](https://github.com/Automattic/jetpack/pull/29108/commits/370a5177214804fa6a9e88c91d19815fe4aece6a)